internal/proposals/logs/unified-approach-to-logging-levels.md

```diff
@@ -63,4 +63,4 @@ There are components that should not be restarted, so this approach couldn't be
 
 ## Setting up the log level in the external components
 
-Currently, we do have a lot of external components that are set up in many ways. We decided that preparing solutions for all of them is out of scope of this proposal.
+Currently, we do have a lot of external components that are set up in many ways. We decided that preparing solutions for all of them is out of scope of this proposal. Those configurations should be described in the documentation, preferrably in a single place, where everyone will have an easy access when needed.
```

Comment:
```suggestion
Currently, we have a lot of external components that are set up in many ways. We decided that preparing solutions for all of them is out of scope of this proposal. Those configurations should be described in the documentation, preferably in a single place, where everyone will have easy access when needed.
```

- The `do` is not necessary here. Its purpose is to add emphasis (e.g. when provided as a counter-argument), which is not needed here. 
- Fixed a typo in `preferably`
- `access` is uncountable, so no `an` here